### ISSUE TITLE
Presentation

### ISSUE THREAD

**Issue by andrew** [andrew/jquery-uk#3]
I've added an initial layout for the presentation: https://github.com/andrew/jquery-uk/blob/master/presentation/Layout.md

Tomorrow I'm going to make the code for the emoji fruit keyboard and the slot machine spinning animation. 

Are you available for a hangout tomorrow or on Monday?

**Comment by andrew** [comment]
I've added the slot machine and the layout sections to the actual slideshow, needs expanding out properly and details filling out but it's getting there.

I've also added some setup instructions to the readme to simplify the running of all the demos and presentation.

Each demo runs on localhost:5x00 which will be shown in an iframe in the slides so we shouldn't need to jump to the console at all in theory.

**Comment by reconbot** [comment]
:heart:

---

Francis Gulotta
wizard@roborooter.com

**Comment by andrew** [comment]
I didn't have as a productive day as I hoped, will double down tomorrow to get as much wrapped up as possible.

I've got the makey makey demo working great (johnny-five doesn't like having two arduino-like devices plugged in at once so we'll have to hardcode the address)

I've added more content to the slides, split them up a bit and left notes for what we still need to fill in (mainly diagrams).

If you get any time to style/arrange some of the list pages to make them bigger/look nicer that would be great.
